### PR TITLE
RHDEVDOCS-365: Adding a new Neon 3 how to

### PR DESCRIPTION
**Pull request by supriya-bharadwaj**
Hi @xcoulon , I want to publish this new Neon 3 article. The article has been reviewed and okay'd by @rawagner . Can you please take care of it? Thanks!

### CODE REVIEW

File: documentation/howto/neon3_getting_started.adoc [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+=== Enabling Bower Init
+After you have enabled Bower Init the result will be a `bower.json` file listed under the project in the *Project Explorer* view.
+
+To enable Bower Init:
+
+. In the *Project Explorer* view, right-click *Neon3_features* and then click *New* > *Other*.
+. In the *New* wizard, *type filter text* field, type _bower_. After *Bower Init* is selected, click *Next*.
+. Optionally, in the *Bower Initialization Wizard*:
+.. Clear the *Use default configuration* check box.
+.. In the *Version* field, type _0.0.1_.
```
Comment (jeffmaury) [inline documentation/howto/neon3_getting_started.adoc]: Version is listed here as 0.0.1 but show as 0.0.0 in the following image

File: documentation/howto/neon3_getting_started.adoc [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+To create an `index.html` file:
+
+. In the *Project Explorer* view, right-click *Neon3_features* and click *New* > *File*.
+. In the *New File* wizard:
+.. Ensure that *Neon3_features* is selected as the parent folder.
+.. In the *File name* field, type _index.html_.
+. Click *Finish*. The empty `index.html` file opens in the default editor.
+. Copy the following code snippet and paste it in the `index.html` file.
++
+[source, java]
```
Comment (jeffmaury) [inline documentation/howto/neon3_getting_started.adoc]: This does not sound like valid HTML and mixed with asciidoc syntax ?

File: documentation/howto/neon3_getting_started.adoc [hunk 3]
```diff
@@ -0,0 +1,371 @@
...
+====
+This section is applicable only if you are using the `package.json` file. Skip this section if you are using the `bower.json` file.
+====
+
+You must add the dependencies to the `package.json` file to be able to use it in <<enable_gulp>>.
+
+To add the dependencies:
+
+. In the *Project Explorer* view, expand *neon3_features* and double-click *package.json* to open the file in the default editor.
+. After *"license": "ISC"*, add a comma (*,*).
```
Comment (jeffmaury) [inline documentation/howto/neon3_getting_started.adoc]: I don't understand, I thought the license was MIT ?

File: documentation/howto/neon3_getting_started.adoc [hunk 4]
```diff
@@ -0,0 +1,371 @@
...
+. On the next line add the following code snippet:
++
+[source, java]
+----
+    "devDependencies" : {
+    "jquery" : "*",
+     "angular" : "*",
+    "bootstrap" : "~3.3.6"
+----
```
Comment (jeffmaury) [inline documentation/howto/neon3_getting_started.adoc]: Aren't we missing a closing bracket here ?

**Comment by supriya-bharadwaj** [comment]
Hi @jeffmaury , I have made all the changes that you have stated above. Can you please have a look at them and let me know of any more edits. Thanks!

**Pull request merged**